omod-1.9/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_9/ConceptResource1_9Test.java

```diff
@@ -34,6 +27,18 @@
 import org.openmrs.module.webservices.rest.web.response.ConversionException;
 import org.openmrs.module.webservices.rest.web.v1_0.resource.openmrs1_8.ConceptResource1_8;
 
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Locale;
+
+import static junit.framework.TestCase.assertFalse;
+import static junit.framework.TestCase.assertTrue;
+import static org.hamcrest.CoreMatchers.*;

```

Comment:
I would add this to the conventions page, yes.

-Darius

On Tue, Jun 7, 2016 at 12:45 PM, dkayiwa <notifications@github.com> wrote:

> In
> omod-1.9/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_9/ConceptResource1_9Test.java
> <https://github.com/openmrs/openmrs-module-webservices.rest/pull/216#discussion_r66139117>
> :
>
> > @@ -34,6 +27,18 @@
> >  import org.openmrs.module.webservices.rest.web.response.ConversionException;
> >  import org.openmrs.module.webservices.rest.web.v1_0.resource.openmrs1_8.ConceptResource1_8;
> >
> > +import java.util.ArrayList;
> > +import java.util.List;
> > +import java.util.Locale;
> > +
> > +import static junit.framework.TestCase.assertFalse;
> > +import static junit.framework.TestCase.assertTrue;
> > +import static org.hamcrest.CoreMatchers.*;
>
> @djazayeri <https://github.com/djazayeri> should i update the conventions
> wiki page to reflect this?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/openmrs/openmrs-module-webservices.rest/pull/216/files/75c040f9c8cc0f8b3498eae7ba6429184399832e#r66139117>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABHHtAyBs_iZeqen4aAkeELUK6oJyawxks5qJcpfgaJpZM4IurrI>
> .
>